dotnetflix.Models/Dtos/TheaterRow/TheaterRowDto.cs

```diff
@@ -4,10 +4,13 @@ public class TheaterRowDto
 {
     public int Id { get; set; }
     
+    public string TheaterName { get; set; }
     public int TheaterId { get; set; }
     
     public int RowNumber { get; set; }
     
+    public int ActualSeatCount { get; set; } 
```

Comment:
ActualSeatCount hoeft niet in de Dto toch, of wel? Je berekent dit op de pagina a.d.h.v. een request?